datastore/save.go

```diff
@@ -138,6 +138,17 @@ func reflectFieldSave(props *[]Property, p Property, name string, opts saveOpts,
 		}
 	}
 
+	if v.CanAddr() {
+		vi := v.Addr().Interface()
+		if pSaver, ok := vi.(PropertyLoadSaver); ok {
+			val, err := pSaver.Save()
+			if err != nil {
+				return fmt.Errorf("field save error: %v", err)
```

Comment:
Should this be prefixed with "datastore: "? I am not sure if that convention is used here or not.

Reply:
As this error is going to be returned by user defined method, I think it shouldn't be prefixed with `datastore`